src/plugins/inputs/away.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+exports.commands = ["away", "back"];
+
+exports.input = function(network, chan, cmd, args) {
+	if (cmd === "away") {
+		let reason = " ";
+
+		if (args.length > 0) {
+			reason = args.join(" ");
+		}
+
+		network.irc.raw("AWAY", reason);
+
+		return;
+	}
+
+	network.irc.raw("AWAY");
```

Comment:
If I understand properly, this will be executed whatever the input. If so, that would act as an auto-unaway so that if I send a message while away, this marks me as not away anymore.
I don't like this too much: I have seen many times people sending messages while away, kind of a "do not disturb" mode. I don't see why we would force the unaway even when user doesn't ask it.

I have been out of the game a couple months, am I misunderstanding something? :)